src/material-experimental/mdc-tabs/harness/tab-group-harness.spec.ts

```diff
@@ -43,6 +43,13 @@ function runTests() {
     expect(tabGroups.length).toBe(1);
   });
 
+  it('should load harness for tab-group with selected tab label', async () => {
```

Comment:
Do we want a test for passing in a regex?

Reply:
done.